packages/react/src/password-input/PasswordInput.tsx

```diff
@@ -0,0 +1,70 @@
+import { forwardRef, InputHTMLAttributes } from 'react';
+import { useId, useToggleState } from '../core';
+import { TextInput } from '../text-input';
+import { Checkbox } from '../checkbox';
+import { Stack } from '../stack';
+
+type NativeInputProps = InputHTMLAttributes<HTMLInputElement>;
+
+type BasePasswordInputProps = {
```

Comment:
could do this with some object mapping magic if desirable.

`type SupportedProps =
  | 'autoComplete'
  | 'autoFocus'
  | 'disabled'
  | 'id'
  | 'maxLength'
  | 'name'
  | 'onBlur'
  | 'onChange'
  | 'onFocus'
  | 'value';

type BasePasswordinputProps = { [K in SupportedProps]?: NativeInputProps[K] };`